### PR TITLE
CPRDIY-914: Upgrade pinpay composer.json to support PHP 7*

### PR DESCRIPTION
**Pull request by indunil15**
@toddhainsworth @carbsrule 
Can you please review this.

### CODE REVIEW

**Comment by toddhainsworth** [comment]
Adding @moloughlin - do we have any projects we might wanna still support PHP 5 on with this module? Given that it's EOL I don't expect so, but better to check :+1:

**Review by toddhainsworth — COMMENTED** [review]
Looks good, just wanna get Matt's thoughts too :)

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "aligent/pinpay",
   "description": "",
   "require": {
-    "php": "~5.5.0|~5.6.0|^7.0.0",
+    "php": "~7.0.0||~7.1.0||~7.2.0||~7.3.0",
```
Comment (moloughlin) [inline composer.json:5]: Interesting; I'd never noticed that `||` is the actual documented separator, good to know :+1:

**Review by moloughlin — APPROVED** [review]

**Comment by moloughlin** [comment]
> Adding @moloughlin - do we have any projects we might wanna still support PHP 5 on with this module? Given that it's EOL I don't expect so, but better to check +1

Nope, we're definitely not running or supporting PHP5 anywhere.

**Pull request merged**